### PR TITLE
Added SVG title element to see also

### LINKED ISSUE #25576: Missing info
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title

### What specific section or headline is this issue about?

missing info about title in SVG and iframe 

### What information was incorrect, unhelpful, or incomplete?

No mention of the legit use of title elements within the page SVG and iframe elements. This is important as there is a misconception out in the wild regarding this exact topic claiming that there must be one and only one title element on the page. This misconception is potentially dangerous to accessibility as SVG title elements are the platform mechanism to convey accessible names for SVGs

### What did you expect to see?

A mention of the legitimacy and importance of the use of title elements within SVG elements

### Do you have any supporting links, references, or citations?

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/title`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/title/index.md
* Last commit: https://github.com/mdn/content/commit/ba96f2f183353872db6d9242c7d2dffe2dbc0c35
* Document last modified: 2023-02-24T05:52:35.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
closes https://github.com/mdn/content/issues/25576

### CODE REVIEW

File: files/en-us/web/html/element/title/index.md [hunk 1]
```diff
@@ -126,3 +126,7 @@ If a form submission contains errors and the submission re-renders the current p
 ## Browser compatibility
 
 {{Compat}}
+
```
Comment (dawei-wang) [inline files/en-us/web/html/element/title/index.md:129]:
```suggestion

```

File: files/en-us/web/html/element/title/index.md [hunk 2]
```diff
@@ -126,3 +126,7 @@ If a form submission contains errors and the submission re-renders the current p
 ## Browser compatibility
 
 {{Compat}}
+  
+## See also
+
```
Comment (dawei-wang) [inline files/en-us/web/html/element/title/index.md:131]:
```suggestion

```

**Review by dawei-wang — COMMENTED** [review]
linting

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**